backend/app/views/spree/layouts/admin.html.erb

```diff
@@ -4,30 +4,43 @@
     <%= render 'spree/admin/shared/head' %>
   </head>
 
-  <body class="admin">
+  <body class="admin <%= "new-layout" if @admin_layout %>">
```

Comment:
okay, maybe I kind of sort of understand. this still seems pretty tricky reading the code. I'd suggest some brief docs explaining how it all works somewhere.